src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -375,17 +408,78 @@ namespace OpenLoco::ObjectManager
 
     // 0x004720EB
     // Returns std::nullopt if not loaded
-    std::optional<uint32_t> getLoadedObjectIndex(const object_index_entry& object)
+    std::optional<LoadedObjectIndex> findIndex(const object_index_entry& object)
     {
-        return getLoadedObjectIndex(object._header);
+        return findIndex(*object._header);
     }
 
     // 0x0047237D
-    void resetLoadedObjects()
+    void reloadAll()
     {
         call(0x0047237D);
     }
 
+    enum class ObjectProcedure
+    {
+        load,
+        unload,
+        proc2,
```

Comment:
think this was validate in RCT2. Checks that refund cost is less than cost to create that sort of thing.